testapp/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,9 @@
               android:name="android.app.searchable"
               android:resource="@xml/searchable" />
         </activity>
-        <activity android:name="com.rakuten.tech.mobile.testapp.ui.display.MiniAppDisplayActivity"/>
+        <activity
+            android:name="com.rakuten.tech.mobile.testapp.ui.display.MiniAppDisplayActivity"
+            android:configChanges="orientation|screenSize"/>
```

Comment:
@corycaywood Would Link require these specific `configChanges` to their respective container activity?